### DISCUSSION TITLE
Is it possible to run just TTS with a portable python version? I got error!

### DISCUSSION THREAD

**Discussion by KStormStudio** [coqui-ai/TTS#4038] [General]
In the main folder TTS-dev I copied the python folder into it.
I just need the text to speech funtion.

Just running:
 python-3.13.0-embed-amd64\python.exe pip install tts

C:\AI Voice Generator\TTS-dev\python-3.13.0-embed-amd64\python.exe: can't open file 'C:\\AI Voice Generator\\TTS-dev\\pip': [Errno 2] No such file or directory

So how can  fix it please?
Many thanks

**Comment by eginhard** [comment]
It should work, but based on the error message you need to install `pip` first, not sure how that works in this portable version. TTS also won't work with Python 3.13. This repo only works up to 3.11, but it's not maintained anymore. I'd recommend to install [our fork](https://github.com/idiap/coqui-ai-TTS) (with `pip install coqui-tts`), which supports up to Python 3.12 for now and has prebuilt wheels for Windows.
